src/algebra/group/pi.lean

```diff
@@ -235,6 +236,20 @@ lemma pi.single_mul [Π i, mul_zero_class $ f i] (i : I) (x y : f i) :
   single i (x * y) = single i x * single i y :=
 (mul_hom.single f i).map_mul x y
 
+/-- The injection into a pi group at different indices commutes.
+
+For injections of commmuting elements at the same index, see `commute.map`
+-/
+@[to_additive]
```

Comment:
Can you `to_additive` the docstring too?

Reply:
Is it just a matter of reversing the order, or do I have to repeat the docstring in the attribute? (Assuming the same text works for both)